### PR TITLE
Added pdf url crawler for KSDMA

### PR DESCRIPTION
**Pull request by arunpt**

### CODE REVIEW

File: dam.js [hunk 1]
```diff
@@ -107,9 +108,11 @@ router.get('/getData', (req, res, next) => {
     }
   }
 
-  download(process.env.DAM_WATER_LEVEL_FILE_LOCATION, "kseb_dam.pdf").then((value) => {
-    pdf_table_extractor("kseb_dam.pdf", success, (error) => on_error(res, error));
-  }, (error) => on_error(res, error));
+  siteCrawler("KSEB").then(link => {
+    download("https://sdma.kerala.gov.in/wp-content/uploads/2021/10/kseb_site_10pm.pdf", "kseb_dam.pdf").then((value) => {
```
Comment (kishorviswanathan) [inline dam.js]: Looks like you forgot to use the `link` variable here.

**Review by kishorviswanathan — COMMENTED** [review]
These changes looks good to me. There is a small comment, other than that it looks fine.

One other suggestion is to keep the pdf in cache and download new one only if necessary. Maybe if you are interested, we can discuss this further at https://t.me/dam_level_app_community

File: dam.js (hunk 1 above)
Comment (arunpt) [reply dam.js]: haha yes 😅

Comment (jgeorge97) [reply dam.js]: @CW4RR10R can you kindly fix this

**Pull request closed without merging**